poncho/src/poncho/package_serverize.py

```diff
@@ -14,226 +14,221 @@
 import ast
 import tarfile
 import hashlib
+import inspect
 
 shebang = "#! /usr/bin/env python3\n\n"
```

Comment:
used anywhere?

Reply:
Inspect is used to get the source code from the library_source_code function and place it into the output Library code file.